pkg/services/pluginsintegration/clientmiddleware/grafana_request_id_header_middleware_test.go

```diff
@@ -85,9 +85,9 @@ func Test_HostedGrafanaACHeaderMiddleware(t *testing.T) {
 		require.Len(t, cdt.CallResourceReq.Headers[GrafanaSignedRequestID], 0)
 	})
 
-	t.Run("Should set Grafana request ID headers if a sanitized data source URL is in the allow list", func(t *testing.T) {
+	t.Run("Should set Grafana request ID headers if a sanitized base data source URL is in the allow list", func(t *testing.T) {
 		cfg := setting.NewCfg()
-		cfg.IPRangeACAllowedURLs = []string{"https://logs.GRAFANA.net/"}
+		cfg.IPRangeACAllowedURLs = []string{"https://logs.GRAFANA.net/some/path"}
```

Comment:
Probably the path should be set not on the config value, but on the URL (line 103).

Reply:
Oh yes, you're right 🙈

Reply:
Btw, I'm a bit confused by these two checks in this test, because the name of the test suggests the headers should be set (line 108 and 109):
```
		require.Len(t, cdt.CallResourceReq.Headers[GrafanaRequestID], 0)
		require.Len(t, cdt.CallResourceReq.Headers[GrafanaSignedRequestID], 0)
```